server/ComputeSuperpixelFeatures/TrainSuperpixelFeatures.xml

```diff
@@ -0,0 +1,218 @@
+      <name>stain_2</name>
+      <label>stain-2</label>
+      <description>Name of stain-2</description>
+      <channel>input</channel>
+      <longflag>stain_2</longflag>
+      <element>hematoxylin</element>
+      <element>eosin</element>
+      <element>dab</element>
+      <element>custom</element>
+      <default>eosin</default>
+    </string-enumeration>
+    <double-vector>
+      <name>stain_2_vector</name>
+      <longflag>stain_2_vector</longflag>
+      <label>stain-2 custom vector</label>
+      <default>-1,-1,-1</default>
+      <description>Custom value for stain-2</description>
+    </double-vector>
+    <string-enumeration>
+      <name>stain_3</name>
+      <label>stain-3</label>
+      <description>Name of stain-3</description>
+      <channel>input</channel>
+      <longflag>stain_3</longflag>
+      <element>hematoxylin</element>
+      <element>eosin</element>
+      <element>dab</element>
+      <element>null</element>
+      <element>custom</element>
+      <default>null</default>
+    </string-enumeration>
+    <double-vector>
+      <name>stain_3_vector</name>
+      <longflag>stain_3_vector</longflag>
+      <label>stain-3 custom vector</label>
+      <default>-1,-1,-1</default>
+      <description>Custom value for stain-3</description>
+    </double-vector>
+  </parameters>
+  <parameters advanced="true">
+    <label>WSI Analysis</label>
+    <description>Whole-slide image analysis (WSI) parameters</description>
+    <double>
+      <name>analysis_tile_size</name>
+      <label>Analysis Tile Size</label>
+      <description>Tile size for blockwise analysis</description>
+      <longflag>analysis_tile_size</longflag>
+      <default>2048</default>
+    </double>
+    <double>
+      <name>analysis_mag</name>
+      <label>Analysis Magnification</label>
+      <description>The magnification at which the analysis should be performed.</description>
+      <longflag>analysis_mag</longflag>
+      <default>5</default>
+    </double>
+    <double>
+      <name>min_fgnd_frac</name>
+      <label>Minimum foreground fraction</label>
+      <description>The minimum amount of foreground that must be present in a tile for it to be analyzed</description>
+      <longflag>min_fgnd_frac</longflag>
+      <default>0.5</default>
+    </double>
+    <region>
+      <name>region</name>
+      <label>Region</label>
+      <longflag>region</longflag>
+      <description>left,top,width,height of the region of interest.  All -1 means the whole image is used.</description>
+      <default>-1,-1,-1,-1</default>
+    </region>
+    <integer>
+      <longflag>maxRegionSize</longflag>
+      <label>Maximum region size</label>
+      <description>Maximum width and height allowed when processing an image, in order to prevent accidentally running on too large a region.  Use -1 for no limit</description>
+      <default>5000</default>
+    </integer>
+  </parameters>
+  <parameters advanced="true">
+    <label>Autoencoder Training</label>
```

Comment:
Added Autoencoder Training label for some args related to autoencoder training and superpixel